src/components/AddAnnonce/Preview/style.jsx

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+const SPreview = styled.div``;
+
+export default SPreview;
```

Comment:
Empty style, I dont't know if it's normal 🤓

Reply:
US taken by Teddy and Victor

Reply:
Style is not an US. When someone takes an US, he/she must take caare of the associated CSS alongside